src/Core.ts

```diff
@@ -1064,14 +1071,21 @@ class Mvvm {
 
 			for (const name of el.getAttributeNames()) {
 				const expression: string = el.getAttribute(name);
-
-				if (name === "data-c-region") {
+				if (name === (ATTRIBUTE_PREFIX + "region")) {
 					const region: Region = this.parent.getRegion(expression);
 					region.setDefaultEl(el as HTMLElement);
 					el.removeAttribute(name);
 				} else if (name.indexOf(EVENT_ATTRIBUTE_PREFIX) === 0) {
 					const eventName: string = name.substr(EVENT_ATTRIBUTE_PREFIX.length);
-					this.addEventDecorator(eventName, expression, el as HTMLElement);
+					try {
+						if (regex.test(eventName)) {
+							this.addEventDecorator(eventName.toLowerCase(), expression, el as HTMLElement);
+						} else {
+							throw new MalformedOnEventError(EVT_NAME_ERR, {'%eventName%': eventName});
+						}
+					} catch (e) {
```

Comment:
Catching something that you immediately throw inside of the try in the same block of code isn't a good idea.  If the call to addEventDecorator() throw an error, it should be guarded separately.  MalformedOnEventError isn't ever being allowed to bubble, which ends having the exception only for logging.

Reply:
Yup... resolved